src/main/java/com/nvs/store/dto/cart/CartItemDto.java

```diff
@@ -0,0 +1,43 @@
+package com.nvs.store.dto.cart;
+
+import com.nvs.store.models.cart.Cart;
+import com.nvs.store.models.product.Product;
+
+public class CartItemDto {
+    private Integer id;
+    private Integer quantity;
+    private Product product;
+
+    public CartItemDto() {
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public Integer getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(Integer quantity) {
+        this.quantity = quantity;
+    }
+
+    public Product getProduct() {
+        return product;
+    }
+
+    public void setProduct(Product product) {
+        this.product = product;
+    }
+
+    public CartItemDto(Cart cart) {
```

Comment:
Move construct at the top, according to best practices the structure is:
static fields 
private fields
constructors 
public methods